src/dc_location.rs

```diff
@@ -24,11 +24,17 @@ pub struct _dc_location {
     pub contact_id: uint32_t,
     pub msg_id: uint32_t,
     pub chat_id: uint32_t,
-    pub marker: *mut libc::c_char,
+    pub marker: Option<String>,
     pub independent: uint32_t,
 }
 
-#[derive(Copy, Clone)]
+impl _dc_location {
+    pub fn new() -> Self {
```

Comment:
no need for `new` if we have `dc_location::new()`

Reply:
I think you mean Default::default?